### ISSUE TITLE
read_id does not exists in single fast5 reads

### ISSUE THREAD

**Issue by usamec** [nanoporetech/ont_fast5_api#31]
Following example from README:

```
with get_fast5_file(fast5_filepath, mode="r") as f5:
        for read in f5.get_reads():
            raw_data = read.get_raw_data()
            print(read.read_id, raw_data)
```

I get an error `AttributeError: 'Fast5File' object has no attribute 'read_id'`

Turns out get_read_id() is correct way in both cases.

**Comment by fbrennen** [comment]
Hi @usamec -- thanks very much for that; it's not clear how it slipped into the documentation, but we have an update coming that should make it work the way it's described.

**Comment by davidnewman02** [comment]
This has been fixed in v3.0
